src/attestation_types/quote.rs

```diff
@@ -19,7 +19,20 @@ pub const INTELVID: [u8; 16] = [
     0x93, 0x9A, 0x72, 0x33, 0xF7, 0x9C, 0x4C, 0xA9, 0x94, 0x0A, 0x0D, 0xB3, 0x95, 0x7F, 0x06, 0x07,
 ];
 
+#[derive(Clone, Debug)]
+/// Error type for Quote module
+pub struct QuoteError(String);
+
+impl fmt::Display for QuoteError {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{:?}", &self.0)
```

Comment:
why output with debug format?

Reply:
No good reason, just habit - feel free to request changes on this kind of thing if it seems odd. : )